app/controllers/chat_controller.rb

```diff
@@ -77,6 +77,8 @@ def disable_chat
   end
 
   def create_message
+    RateLimiter.new(current_user, "create_chat_message", 1, 1.second).performed! # 1 message per second
```

Comment:
Can we add an rspec test this as well?

Reply:
Yeah I guess! It feels a bit like testing the implementation, since `RateLimiter#perform` is tested, but it can't hurt

Reply:
Ya I like it more for regression testing. So that any future changes we make to the controller we know we didn't accidentally remove the rate limiter.